rali/rali_pybind/example/coco_pipeline.py

```diff
@@ -39,22 +33,30 @@ def __init__(self, batch_size, num_threads, device_id, data_dir,ann_dir, crop, r
 													random_area=[0.1, 1.0],
 													num_attempts=100)
 		self.res = ops.Resize(device=rali_device, resize_x=crop, resize_y=crop)
+		self.twist = ops.ColorTwist(device=rali_device)
 		self.cmnp = ops.CropMirrorNormalize(device="gpu",
 											output_dtype=types.FLOAT,
 											output_layout=types.NCHW,
 											crop=(crop, crop),
 											image_type=types.RGB,
-											mirror=1,
+											mirror=0,
 											mean=[0.485 * 255,0.456 * 255,0.406 * 255],
 											std=[0.229 * 255,0.224 * 255,0.225 * 255])
-		self.coin = ops.CoinFlip(probability=0.5)
+		# Random variables
+		self.rng1 = ops.Uniform(range=[0.5, 1.5])
+		self.rng2 = ops.Uniform(range=[0.875, 1.125])
+		self.rng3 = ops.Uniform(range=[-0.5, 0.5])
 		print('rali "{0}" variant'.format(rali_device))
 
 	def define_graph(self):
-		rng = self.coin()
+		saturation = self.rng1()
+		contrast = self.rng1()
+		brightness = self.rng2()
+		hue = self.rng3()
 		self.jpegs,self.bb, self.labels = self.input(name="Reader")
 		images = self.decode(self.jpegs)
 		images = self.res(images)
+		images = self.twist(images, saturation=saturation, contrast=contrast, brightness=brightness, hue=hue)
```

Comment:
Make sure saturation, contrast, and brightness are different for each image in images

Reply:
The values are different for each images and is within the range we set in Uniform.